File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Reports/ReportBuilderProjectionModel.ts
import ReportBuilderPropertyModel = require('App/Models/Reports/ReportBuilderPropertyModel');
import ReportBuilderProjectionType = require('App/Models/Reports/ReportBuilderProjectionType');

class ReportBuilderProjectionModel {
    constructor() {
        this.ReportProperties = [];
    }

    ReportProperties: ReportBuilderPropertyModel[];
    ProjectionType: ReportBuilderProjectionType;
    Alias: string;
    TagName: string;
}

export = ReportBuilderProjectionModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Reports/Implementation/ReportBuilderProjectionModel.cs
using System.Collections.Generic;

namespace VecompSoftware.DocSuiteWeb.Model.Reports
{
    public class ReportBuilderProjectionModel
    {
        #region [ Constructor ]
        public ReportBuilderProjectionModel()
        {
            ReportProperties = new List<ReportBuilderPropertyModel>();
        }
        #endregion

        #region [ Properties ]
        public ICollection<ReportBuilderPropertyModel> ReportProperties { get; set; }
        public ReportBuilderProjectionType ProjectionType { get; set; }
        public string Alias { get; set; }
        public string TagName { get; set; }
        #endregion
    }
}
